Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -114,16 +123,19 @@ upload_files () {
         ;;
     esac
 
-    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; }
-
+    eval "$cmd" || { echo "(ERROR) An error occured while uploading file: $INPUTFILE" >&2; continue; } &
+    echo -ne EOMessage
```

Comment:
EOMessage here looks a bit excessive -- Output Connector Stage (here -- Stage 60) sends no data back to supervisor program, so there`s no messages to be separated.